container/client_test.go

```diff
@@ -1,44 +1,5 @@
 package container
 
-import (
-	"testing"
-
-	"github.com/stvp/assert"
-)
-
-func TestClient(t *testing.T) {
-	client, err := Client()
-	assert.Nil(t, err)
-	assert.NotNil(t, client)
-}
-
-func TestClientIsTheSame(t *testing.T) {
-	client, err := Client()
-	assert.Nil(t, err)
-	assert.NotNil(t, client)
-	client2, err := Client()
-	assert.Nil(t, err)
-	assert.NotNil(t, client2)
-	assert.Equal(t, client, client2)
-}
-
-func TestClientNotIsTheSame(t *testing.T) {
-	client, err := Client()
-	assert.Nil(t, err)
-	assert.NotNil(t, client)
-	resetClient()
-	client2, err := Client()
-	assert.Nil(t, err)
-	assert.NotNil(t, client2)
-	assert.NotEqual(t, client, client2)
-}
-
-func TestCreateSwarmIfNeeded(t *testing.T) {
-	client, _ := createClient()
-	err := createSwarmIfNeeded(client)
-	assert.Nil(t, err)
-}
-
 // TODO: this tests break other tests on my machine
```

Comment:
Let's remove completely this file. I don't think we really need integration test on the `createSwarm` functionality. 
@antho1404 what do you think?

Reply:
Why would you want to remove it ? if the test is ok let's keep it, after I agree swarm tests are not  absolutely necessary but I think it's still really good to keep these tests except if there is a good reason to remove them

Reply:
https://github.com/mesg-foundation/core/blob/e0e344e6449caabd857f0eebaa7eb81e99236309/container/client_test.go#L3-L25

Reply:
my bad you're right

Reply:
I'm removing `core/container/client_test.go` file.